ts/image-picker/image-picker.directive.ts

```diff
@@ -11,6 +11,7 @@ module lui.imagepicker {
 			croppingDisabled: "=",
 			deleteEnabled: "=",
 			hideEditHint: "=",
+			disabled: "&"

```

Comment:
dont use `disabled` as attribute cuz if ie sees it, it shuts down all js events for its children

use `isDisabled` instead